p2p/chain.py

```diff
@@ -57,9 +55,12 @@
 if TYPE_CHECKING:
     from trinity.db.chain import AsyncChainDB  # noqa: F401
     from trinity.db.header import AsyncHeaderDB  # noqa: F401
+    from trinity.protocol.eth.peer import ETHPeer  # noqa: F401
+    from trinity.protocol.les.peer import LESPeer  # noqa: F401
+    from trinity.protocol.base_request import BaseHeaderRequest  # noqa: F401
```

Comment:
The goal is to cut all dependencies from p2p to trinity, right? Maybe all of p2p/chain needs to move to trinity. Or... and I hesitate to say it... should there be an eth-p2p?

Reply:
This whole module will get moved, probably in the next wave.